core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -256,6 +259,10 @@ final Handler configureHandler() {
       context.addFilter(filterHolder, "/*", EnumSet.of(DispatcherType.REQUEST));
     }
 
+    if (!config.getString(RestConfig.RESPONSE_HTTP_HEADERS_CONFIG).isEmpty()) {
```

Comment:
to be consistent with other config usage, maybe we can also add null check. (similar to line no:251)